src/test/java/JdiHomework/main/utils/CredentialsManager.java

```diff
@@ -0,0 +1,64 @@
+package JdiHomework.main.utils;
+
+import org.w3c.dom.Document;
+import org.w3c.dom.Element;
+import org.xml.sax.SAXException;
+
+import javax.xml.parsers.DocumentBuilder;
+import javax.xml.parsers.DocumentBuilderFactory;
+import javax.xml.parsers.ParserConfigurationException;
+import javax.xml.xpath.XPath;
+import javax.xml.xpath.XPathConstants;
+import javax.xml.xpath.XPathExpressionException;
+import javax.xml.xpath.XPathFactory;
+import java.io.File;
+import java.io.IOException;
+
+public class CredentialsManager {
+
+    static Document doc;
+    static XPath xPath = XPathFactory.newInstance().newXPath();
+
+    static {
+        try {
+            File inputFile = new File("src/test/resources/credentials.xml");
+            DocumentBuilderFactory dbFactory = DocumentBuilderFactory.newInstance();
+            DocumentBuilder dBuilder;
+
+            dBuilder = dbFactory.newDocumentBuilder();
+
+            doc = dBuilder.parse(inputFile);
+            doc.getDocumentElement().normalize();
+        } catch (ParserConfigurationException e) {
```

Comment:
What is the reason of several catch blocks? You don't you create ony one with base exception ?